UnityProject/Assets/Scripts/Tool Construction Deconstruction/Tool.cs

```diff
@@ -0,0 +1,36 @@
+
+using UnityEngine;
+
+/// <summary>
+/// Component that indicates an object can be used as a tool. Various other components
+/// have interaction logic that checks if a certain kind of tool is being used on them.
```

Comment:
YAY

Reply:
~~quickly no one tell him that's just from a different script it's been moved and had some stuff changed in it~~ : P